mesonbuild/modules/cuda.py

```diff
@@ -325,7 +339,8 @@ def _nvcc_arch_flags(self, cuda_version, cuda_arch_list='Auto', detected=''):
 
             if version_compare(arch, '<' + cuda_lo_limit_gpu_architecture):
                 continue
-            if version_compare(arch, '>=' + cuda_hi_limit_gpu_architecture):
+            if cuda_hi_limit_gpu_architecture and \
+               version_compare(arch, '>=' + cuda_hi_limit_gpu_architecture):
```

Comment:
This is a strange indentation.

Reply:
What was the issue with leaving it on one line like it was before?

Reply:
@tristan957 @eli-schwartz To emphasize the similarity of this `version_compare()` to that of the previous line, while eliminating the `cuda_lo_limit_gpu_architecture and` as previously requested, I arranged to set both `version_compare()`s on exactly the same level. That is the purpose of the `and \`.

Reply:
@eli-schwartz To be clear, I'd prefer to keep this comparison as-is, as I consider it more readable. But if you tell me to put it back on one line and damn the parallels with the previous `version_compare()`, I will do that.

Reply:
It's probably a bit of a personal preference. I myself find it more readable to do it on one line. Plus, a three space indent is quite inconsistent with the rest of the codebase (and pep8, really).

So yes I think I'd prefer changing this.